langchain4j-core/src/test/java/dev/langchain4j/store/embedding/EmbeddingStoreWithoutMetadataIT.java

```diff
@@ -356,7 +356,13 @@ void should_return_correct_score() {
                 .build()).matches()).isEqualTo(relevant);
     }
 
+    @Deprecated
```

Comment:
Could you please explain why?

Reply:
I extracted this to a static method, to use in a test class that did not extend this class. It seems like a generally useful utility method.

I added this to discourage use of this variant, since it seemed only necessary for a workaround. Looking closer, when Awaitility was added, it did not replace the sleep, but was [added before the sleep](https://github.com/langchain4j/langchain4j/pull/1610/files#diff-4c1e931865cee83990f37441139cbc2b0052a06b818f9d0396a9909f7a702dd7R326). I believe the sleep is no longer needed, but I am not set up to test Azure. I could add the following [commit](https://github.com/langchain4j/langchain4j/commit/99eddce0dcaa9c894acfa6490461bc1dcaf48dee) (removal of extra sleep) to this PR, or I could get rid of the Deprecated, or I could leave it. Let me know what you think.

Reply:
@katcharov oh nice, it seems that sleep hack is not needed any more for Azure (just tested), so please feel free to apply your commit

Reply:
Applied